### ISSUE TITLE
[Linux] Click on any account icon, Twitter login will open

### ISSUE THREAD

**Issue by neilsf** [eul-im/eul#85]
After launching program, the "add an account" window opens. Whichever icon you click, it will open the Twitter login window (in two instances).

Now close the program and run again - the icons don't respond at all. Log message:

`curl_easy_perform() failed: Problem with the SSL CA cert (path? access rights?)`

Program version: 0.34

System:
elementary OS 0.4.1 Loki 64-bit
Built on "Ubuntu 16.04.3 LTS"
Linux 4.13.0-38-generic
GTK+ 3.18.9

**Comment by bogdan-radocea** [comment]
I think that the client wants to download some resources from eul.im, which has no DNS entries.
```
user@host:~/eul$ sudo ./eul_linux 
eul 0.34 \n
== 1 49
curl_easy_perform() failed: Couldn't resolve host name
/home/user/.eul/euldata0.34/
initial setup...
curl_easy_perform() failed: Couldn't resolve host name
done!
2CREATE win gl=0 title=eul v0.34
2CREATE win gl=0 title=Add an account
RUNNING GTK MAIN
curl_easy_perform() failed: Couldn't resolve host name
```

**Comment by voltapp2** [comment]
Fixed in 0.35
